client/src/api/mapStoreToLocal.js

```diff
@@ -0,0 +1,5 @@
+// temporary solution
+export default function forwardToken(token) {
```

Comment:
rename this function to be saveTokenToLocalStorage

Reply:
Done
https://github.com/Levayv/ipma_v2/commit/93b21257a3ebd029bd1863aac55fb6c89ad42eac